src/assisted_test_infra/download_logs/__main__.py

```diff
@@ -56,7 +56,6 @@ def main():
             args.dest,
             args.must_gather,
             args.update_by_events,
-            pull_secret=args.pull_secret,
```

Comment:
This field is unused?
If so, can you include explanation for those changes in PR description?

Reply:
It's unused. During this change I noticed that the pull_secret is grayed out so I removed it.
I'll add it to the PR description.